examples/autograd/singa-onnx/mplonnx2singa.py

```diff
@@ -0,0 +1,75 @@
+from singa import tensor
+from singa.tensor import Tensor
+from singa import autograd
+from singa import optimizer
+import numpy as np
+import onnx
+from onnx import numpy_helper
+from onnx_model import *
```

Comment:
onnx and onxx_model are two python packages?

Reply:
I merge them to single file sonnx.py(short for singa-onnx)
onnx_model is my utils file, I delete it now.
onnx is official library